app/lib/searching.rb

```diff
@@ -14,8 +29,13 @@ def count
   def call
     rummager = GdsApi::Rummager.new(Plek.new.find('rummager'))
     fields = %w(
-      title link public_timestamp description format
-      )
+      description
+      format
+      link
+      public_timestamp
+      title
+      content_id
+    ) + ENHANCED_FIELDS + HEAD_FIELDS + SECONDARY_HEAD_FIELDS
```

Comment:
This list of fields also feels like a constant?

Reply:
agreed, done